packages/fxa-content-server/server/lib/routes/react-app/index.js

```diff
@@ -70,31 +82,37 @@ const getReactRouteGroups = (showReactApp, reactRoute) => {
         'signup_verified',
         'oauth/signup',
       ]),
+      fullProdRollout: false,
     },
 
     pairRoutes: {
       featureFlagOn: showReactApp.pairRoutes,
       routes: [],
+      fullProdRollout: false,
     },
 
     postVerifyOtherRoutes: {
       featureFlagOn: showReactApp.postVerifyOtherRoutes,
       routes: [],
+      fullProdRollout: false,
     },
 
     postVerifyCADViaQRRoutes: {
       featureFlagOn: showReactApp.postVerifyCADViaQRRoutes,
       routes: [],
+      fullProdRollout: false,
     },
 
     postVerifyThirdPartyAuthRoutes: {
       featureFlagOn: showReactApp.postVerifyThirdPartyAuthRoutes,
       routes: reactRoute.getRoutes(['post_verify/third_party_auth/callback']),
+      fullProdRollout: false,
     },
 
     webChannelExampleRoutes: {
       featureFlagOn: showReactApp.webChannelExampleRoutes,
       routes: reactRoute.getRoutes(['web_channel_example']),
+      fullProdRollout: false,
```

Comment:
Q: what happens if we forget to put this prop for new route groups? Should this just be defaulted to `false` somewhere higher to avoid needing to set these `false` in 8+ places?

Reply:
Good call. Though I don't think we'll be adding more routes, I've changed the check and added a couple of test cases to make sure `showReactApp` works as expected if `fullProdRollout` is undefined.

Reply:
*me shakes fist at JS Doc*